tests/e2e/contracts/multi_file_contract/other.py

```
@@ -0,0 +1,14 @@
+# { "Depends": "py-genlayer:test" }
+
+from genlayer import *
```

Ruff (0.11.9)

`from genlayer import *` used; unable to detect undefined names

(F403)